packages/react/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "react",
```

Comment:
A name like "react" is cannot be published to npm. This has to be a unique name. We need to support scoped names. So ideally something like `@code-shaper/react`. But when I try this, getPlugins() does not detect it, because I think it goes only 1 level deep into npm_modules. You need to go at least 2 levels deep. How do other people do this?

To do deep tree traversals, you should use `klaw-sync`. See FileUtils